src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -2899,23 +2899,37 @@ EmberAfStatus DoorLockServer::clearCredentials(chip::EndpointId endpointId, chip
         return EMBER_ZCL_STATUS_FAILURE;
     }
 
+    EmberAfStatus status   = EMBER_ZCL_STATUS_SUCCESS;
+    bool clearedCredential = false;
     for (uint16_t i = 1; i < maxNumberOfCredentials; ++i)
     {
-        auto status = clearCredential(endpointId, modifier, sourceNodeId, credentialType, i, false);
-        if (EMBER_ZCL_STATUS_SUCCESS != status)
+        EmberAfStatus clearStatus = clearCredential(endpointId, modifier, sourceNodeId, credentialType, i, false);
```

Comment:
that `false` should at least have a comment saying `/* sendUserChangeEvent */` or similar. Or better yet not be a bool and be `UserSendEvent::kDoNotSend` or similar.

Not part of this PR, just complaining that reviewing code is harder when such bools are used as arguments.

Reply:
Yes, there's a lot of cleanup we could try to do here...  :(